dev/com.ibm.ws.webcontainer.cors/src/com/ibm/ws/webcontainer/cors/CorsHelper.java

```diff
@@ -67,7 +67,7 @@
     private static final String RESPONSE_HEADER_ACCESS_CONTROL_EXPOSE_HEADERS = "Access-Control-Expose-Headers";
 
     /** Optional. Response to preFlight(OPTIONS). Indicates how long the results of a preFlight request can be cached. */
-    private static final String RESPONSE_HEADER_ACCESS_CONTROL_MAX_AGE = "Access-Control-Allow-Max-Age";
```

Comment:
Please update the [copyright header](https://github.com/OpenLiberty/open-liberty/pull/9952/files#diff-1c806706d819c2cd9f8ff364d77a9133R2) in `Copyright (c) 2015, 2017...` to `Copyright (c) 2015, 2019...`